src/rng.cpp

```diff
@@ -1,19 +1,30 @@
-#include "output.h"
 #include "rng.h"
-#include <stdlib.h>
+#include <random>
 
-long rng(long val1, long val2)
+namespace {
+std::mt19937 random_generator {std::random_device{}()};
```

Comment:
Yea, consistency is a complete non-issue, we don't have anything remotely resembling repeatability anyway, since even with the same seed, user inputs will inject more entropy into the system and cause different code paths and different sequences of rng calls to happen, scrambling the game state very quickly.